### PR TITLE
app-admin/yadm: add 3.2.1

### PR DESCRIPTION
**Pull request by ixti**
Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Alexey Zapparov <alexey@zapparov.com>

### CODE REVIEW

**Comment by ixti** [comment]
@gyakovlev please review when you have a time. I've added `DOCS` since last bump:

``` patch
36a37,38
> DOCS=(CHANGES README.md ${PN}.md)
> 
```

File: app-admin/yadm/yadm-3.2.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+			dev-python/pytest[${PYTHON_USEDEP}]
+		')
+		dev-tcltk/expect
+	)
+"
+
+DOCS=(CHANGES README.md ${PN}.md)
+
+python_check_deps() {
+	has_version -b "dev-python/pytest[${PYTHON_USEDEP}]"
```
Comment (arthurzam) [inline app-admin/yadm/yadm-3.2.1.ebuild:40]: Please use the new python way :)
```suggestion
	python_has_version -b "dev-python/pytest[${PYTHON_USEDEP}]"
```

File: app-admin/yadm/yadm-3.2.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+src_test() {
+	# prevent system config having influence on tests
+	local -x GIT_CONFIG_NOSYSTEM=1
+
+	# prevent git branch warning, it confuses tests
+	local -x GIT_CONFIG_GLOBAL="${HOME}/.gitconfig"
+	git config --global init.defaultBranch master || die "setting default branch name failed"
+
+	# un-hardcode path to tmp, otherwise encryption tests fail
+	sed -i "s@^cache_dir = /tmp@cache_dir = ${TMPDIR}@" pytest.ini || die
```
Comment (arthurzam) [inline app-admin/yadm/yadm-3.2.1.ebuild:60]: We can improve a little the `sed`:
```suggestion
	sed -e "/^cache_dir/s@/tmp@${T}@" -i pytest.ini || die
```

About the tmpdir change. Based on [pms](https://projects.gentoo.org/pms/8/pms.html#x1-10900011.1), it is preferable to use `${T}` over `${TEMPDIR}`

**Review by arthurzam — CHANGES_REQUESTED** [review]
Some small changes :)

File: app-admin/yadm/yadm-3.2.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+		${RDEPEND}
+		${PYTHON_DEPS}
+		$(python_gen_any_dep '
+			dev-python/pytest[${PYTHON_USEDEP}]
+		')
+		dev-tcltk/expect
+	)
+"
+
+DOCS=(CHANGES README.md ${PN}.md)
```
Comment (arthurzam) [inline app-admin/yadm/yadm-3.2.1.ebuild:37]: Please with spaces after & before braces

```suggestion
DOCS=( CHANGES README.md ${PN}.md )
```

**Comment by ixti** [comment]
@arthurzam thank you for the review. Applied your proposals.

**Comment by gyakovlev** [comment]
@arthurzam thanks for working on it ( and couple other PRs too)
@ixti and thanks for bump. I was away, usually I react faster.

**Comment by ixti** [comment]
> I was away, usually I react faster.

There was no rush. :D

**Pull request closed without merging**